docs/features/software-catalog/well-known-annotations.md

```diff
@@ -146,6 +146,22 @@ repository itself. If the URL points to a folder, it is important that it is
 suffixed with a `'/'` in order for relative path resolution to work
 consistently.
 
+### backstage.io/source-template
+
+```yaml
+# Example:
+metadata:
+  annotations:
+    backstage.io/source-template: template:default/create-react-app-template
```

Comment:
🔥

Reply:
How does this get added? Can it be done automatically by scaffolder?

Reply:
Only viable path in my mind is for it to be encoded in the underlying template by the owner of the Template (e.g. in cases where the template includes a `catalog-info.yaml` file that gets registered).  Can't really think of a viable way for those whose entities are ingested by non-declarative means.

Reply:
@benjdlambert sayz: maybe the catalog write action could be a spot to help automate this.